### DISCUSSION TITLE
content/posts/youtubeafreecacrawling/

### DISCUSSION THREAD

**Discussion by giscus** [cha2hyun/cha2hyun.github.io#105] [Comments]
# content/posts/youtubeafreecacrawling/

feat.배돌이의 당구생활) 멀티쓰레딩 + 실행파일로 생성

https://cha2hyun.blog/content/posts/youtubeafreecacrawling/

**Comment by peachapplebear** [comment]
안녕하세요, 유튜브 실시간채팅 크롤링을 검색해서 들어온 비개발자 입니다
저는 유튜브 실시간 채팅창 크롤링 후->댓글창을 통한 투표 진행->통계를 위한 집계 후 결과 발표 
프로그램을 구상하고 있는데요, 
채수현님의 포스팅이 많은 도움이 되었습니다!! 감사합니다 :)
혹시 괜찮으시다면 코딩에 대해 궁금한 점을 댓글이나 메일을 통해 더 여쭤볼 수 있을까요?

**Reply by cha2hyun** [reply]
안녕하세요 @peachapplebear 님 

네 내용 남겨주시면 최대한 확인해보겠습니다 :)
